data/campaigns/Two_Brothers/scenarios/03_Guarded_Castle.cfg

```diff
@@ -323,6 +323,40 @@ Yet for some reason I fear these brothers more. If Mordak were here it would be
                 bonus=yes
                 carryover_percentage=40
             [/gold_carryover]
+
+#ifdef EASY
+            {HINT ( _ "When facing an unknown situation, take into account details from story and dialog to inform your strategy.") (
+                [show_if]
+                    [not]
+                        [have_unit]
+                            id=Baran
+                        [/have_unit]
+                    [/not]
+                [/show_if]
+            )}
+            {HINT ( _ "Lure the dark sorcerer off of his keep by giving him a shot at a melee unit, then counter-attack in force.") (
```

Comment:
This hint just tells the player how to exploit an AI shortsightedness, doesn't it?  I.e., if a human player controlled the dark sorcerer, this strategy wouldn't work.

(I'd _love_ to be able to play the mainline campaigns with humans controlling the enemy sides, by the way.)

Reply:
The idea of these hints is to introduce new players to the game. Yes, it's an AI weakness, and no human would fall for the trick. So what? We want the new player to succeed. If that means showing them the "AI" aint that "I" then so be it.,

Reply:
@GregoryLundberg I think we want new players to succeed even against strong opponents.  How about rephrasing the advice to be more generic, e.g., "The dark sorcerer is a strong unit; attack him with multiple units at once and try to force him off of his keep."?